odk1-src/validate.rst

```diff
@@ -59,14 +56,15 @@ GUI
 
    .. image:: /img/validate/validform.png
 
-Command Line
+Command line
 ~~~~~~~~~~~~
 
-``java -jar ODKValidate.jar [--fail-fast] path/to/xform.xml [FORM...]``. The
-``--failFast/--fail-fast`` command line argument tells the validator to abort
-on the first error rather than validating all files and reporting an error at
-the end.
-  
+.. code-block:: console
+
+  $ java -jar {path/to/validate-jar-file} [--fail-fast] {path/to/xform1.xml} {path/to/xform2.xml}
```

Comment:
How about making it clearer that an arbitrary number of forms can be specified? I think there should be some text to describe that.

Reply:
Change made and force pushed. If you are happy with the changes, please review and merge!